nixos/modules/services/misc/mesos-slave.nix

```diff
@@ -43,6 +51,12 @@ in {
         type = types.bool;
       };
 
+      withDocker = mkOption {
```

Comment:
@ragnard Thanks! This looks good to me as well. @wmertens @offlinehacker May we please have this merged?